docs/src/docs/guielement/guielement-creation.adoc

```diff
@@ -29,6 +31,23 @@ GuiElement lower = upper.getSubElement(By.xpath("./p[@id='element']")); <3>
 <2> Corrects the selector prefix to './/'
 <3> Find any matching child
 
+== Element lists
+
+A locator can always return multiple elements. Even if a GuiElement is just a pointer to the first element of the result, it could contain more than one.
+
+You can get the amount of found elements by calling.
+[source,java]
+----
+GuiElement span = new GuiElement(driver, By.xpath("//a/span"));
+int count = span.getNumberOfFoundElements();        // returns 2
+----
+
+And access the elements by using the `getList()` method.
+[source,java]
+----
+List<GuiElement> allSpans = span.getList();
+----
+
```

Comment:
Should added:
If you iterate over the list, every GuiElement points to one unique web element based on a internal index. (or similar)